src/UIAutomation/FunctionalTests/UiaOperationAbstractionTests.cpp

```diff
@@ -479,5 +479,58 @@ namespace UiaOperationAbstractionTests
         {
             ArrayEqualityComparisonTest(true);
         }
+
+        void ForEachLoop(bool useRemoteOperations)
+        {
+            // Initialize the test application.
+            ModernApp app(L"Microsoft.WindowsCalculator_8wekyb3d8bbwe!App");
+            app.Activate();
+
+            // Set focus to the display element.
+            auto focusedElement = WaitForElementFocus(L"Display is 0");
+
+            // Initialize the UIA Remote Operation abstraction.
+            const auto cleanup = InitializeUiaOperationAbstraction(useRemoteOperations);
+
+            {
+                auto operationScope = UiaOperationScope::StartNew();
```

Comment:
If you feel strongly about breaking this up into 2-3 independent tests for more functionality-isolated coverage, I can definitely do it.

Reply:
looks good :)